pepper-apis/dss-database/src/main/resources/db-changes/schema/DDP-8511-physician-country-add.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="gskhirel" id="physician_country_add">
+        <addColumn tableName="user_medical_provider">
+            <column name="country" type="varchar(100)">
```

Comment:
wondering if it should be country or country_id (reference to country) like we have in mailing_address .
I think country should be fine though. Everything else looks to me.
@MocanaAtBroad : any thoughts on this ?

Reply:
I would encourage you to use country_address_info table. Has all the countries in the world and we have translations for them. You can also do autocompletion.